### ISSUE TITLE
conversion of .mpk saves question

### ISSUE THREAD

**Issue by FrostKnight** [fgsfdsfgs/perfect_dark#520]
What do I need to name the file, for my save to be loadable in this project?

I have many things saved in it and I would like to be able to use them.

**Comment by fgsfdsfgs** [comment]
There is a [save converter](https://github.com/MaikelChan/PDSaveEditor). You can probably convert the MPK save with it.

**Comment by FrostKnight** [comment]
> There is a [save converter](https://github.com/MaikelChan/PDSaveEditor). You can probably convert the MPK save with it.

That's true, but I don't know how to convert it with that. Can you test such a method yourself? I tried most options including that one and it failed.

Also, I don't know where save files go for your application and what they are named, etc...

**Comment by FrostKnight** [comment]
Question though, what is the rom save called when you save the game?

I don't have any clue you see...

is it eeprom.bin or something else?  What should the save be named?

**Comment by MaikelChan** [comment]
Hi!

My save editor / converter doesn't support controller paks, as that requires dealing with a bunch of extra stuff. What you could do is:

- Load the N64 game in an emulator, and use the built in functionality to copy the data you need from the controller pak to the cartridge.
- You should have your data in a `.eep` file that you can open with my editor.
- In my editor, go to the menu `Tools` and change the `Save Format` from N64 to PC.
- Go to the menu `File/Save As...` and save it with the name `eeprom.bin` along with the PC port executable.

Your data now should be loaded correctly on PC.

**Comment by meldridge** [comment]
@MaikelChan I tried to follow this process, and I successfully moved all of the savegames/simulator into the cart save (.eep), but it refuses to open with your editor. My .eep is 4KB, and it sounds like your converter has a 2KB limit? Is there a way to skip this check?

If it helps, mine is from the PAL version, it loads fine in the 1964 emulator, but I'd really like to play it on the PC port.

EDIT: Nevermind, found [this issue](https://github.com/MaikelChan/PDSaveEditor/issues/1) and that fixed it. Thanks for this!
